.gitignore

```diff
@@ -4,3 +4,4 @@ target/
 .project
 .settings/
 .idea/
+*.iml
```

Comment:
Some IntelliJ project file extension. I'm generating them to import the project, but without `.gitignore`ing it my `git status` and `git diff` become messy.